subnet_router.go

```diff
@@ -0,0 +1,99 @@
+package packngo
+
+import (
+	"path"
+)
+
+const subnetRouterBasePath = "/subnet-routers"
+
+// DevicePortService handles operations on a port which belongs to a particular device
```

Comment:
comment block has the wrong description

I think we should call this `SubnetRouterService` instead of `ProjectSubnetRouterService`. Is there any reason why this should be `Project*` instead? Many of our resources are project scoped but don't include a `Project` prefix.

Reply:
I think the reason was that the vlan and ip service name has Project prefix. But I'll remove it.

Well spotted with the comment.